src/index.ts

```diff
@@ -185,7 +192,7 @@ export class VuexPersistence<S> implements PersistOptions<S> {
             store.replaceState(merge(store.state, savedState || {}, this.mergeOption) as S)
           }
           this.subscriber(store)((mutation: MutationPayload, state: S) => {
-            if (this.filter(mutation)) {
+            if (!this.shuttingDown && this.filter(mutation)) {
```

Comment:
I don't think we should stop accepting writes here. We can keep accepting more writes and `prepareShutdown` will still wait for those writes to finish too.

Reply:
I don't think we should plan around that. If we're making so many mutations we can't flush our write queue we have bigger problems because we will never persist data in the real world data either. For example, we could consider a custom saveState function in that case that debounces or something.